recipes/bson/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bson" %}
+{% set version = "0.4.8" %}
+{% set sha256 = "6cea0806b4b85ab26f4241afb8e0f7e82232cdac52447bb3267b16fa975ef53a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - six
+    - pytz
+
+test:
+  imports:
+    - bson
+
+about:
+  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
+  home: https://github.com/py-bson/bson
+  license: BSD-3-Clause
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+    - robmcmullen
```

Comment:
@robmcmullen -- are you ok being added as a co-maintainer of the `bson` package on conda-forge?